src/hdmf/common/resources.py

```diff
@@ -82,21 +89,35 @@ class ObjectTable(Table):
     A table for storing objects (i.e. Containers) that contain keys that refer to external resources.
     """
 
-    __defaultname__ = 'objects'
+    __defaultname__ = "objects"
 
     __columns__ = (
-        {'name': 'files_idx', 'type': int,
-         'doc': 'The row idx for the file_object_id in FileTable containing the object.'},
-        {'name': 'object_id', 'type': str,
-         'doc': 'The object ID for the Container/Data.'},
-        {'name': 'object_type', 'type': str,
-         'doc': 'The type of the object. This is also the parent in relative_path.'},
-        {'name': 'relative_path', 'type': str,
-         'doc': ('The relative_path of the attribute of the object that uses ',
-                 'an external resource reference key. Use an empty string if not applicable.')},
-        {'name': 'field', 'type': str,
-         'doc': ('The field of the compound data type using an external resource. '
-                 'Use an empty string if not applicable.')}
+        {
+            "name": "files_idx",
+            "type": int,
+            "doc": "The row idx for the file_object_id in FileTable containing the object.",
+        },
+        {"name": "object_id", "type": str, "doc": "The object ID for the Container/Data."},
```

Comment:
I think this should also follow the style changes of the others.

Reply:
I agree that this is ugly. I'll look into what I can change to make this consistent.